### PR TITLE
Fix features relying on `.pagehead` 

### LINKED ISSUE #3367: `ci-link`: Cannot read property 'append' of null
`ci-link` is not visible on any page where it's supposed to be.

Screenshot 2020-07-18 20 35 50

Also getting a bunch of other errors along with this error:

Screenshot 2020-07-18 20 36 05

### PR DESCRIPTION
**Pull request by yakov116**
Selector Update

Fixes #3367

### CODE REVIEW

**Comment by fregante** [comment]
Wait, did they remove the pagehead class? If so, it should be replaced in a bunch of features

**Comment by yakov116** [comment]
@fregante  yes and it's not so simple

**Comment by kidonng** [comment]
> Wait, did they remove the pagehead class? If so, it should be replaced in a bunch of features

`.gisthead` and `.pagehead` still exists on Gits; `.pagehead` and `.repohead` is gone on main site.

**Comment by fregante** [comment]
That class is not unique enough, it’s a utility class

**Comment by yakov116** [comment]
I dont see anything else 😞. Its all generic classes

**Comment by yakov116** [comment]
This is not going to be fun. There are many more features that are broken.

**Comment by yakov116** [comment]
I fixed some more, I did not add any comments (yet). I really dont see anything we can work with

File: source/features/ci-link.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ async function init(): Promise<false | void> {
 	}
 
 	// Append to title (aware of forks and private repos)
-	appendBefore('.pagehead h1', '.fork-flag', icon);
+	appendBefore('.pagehead h1, .hide-full-screen h1', '.fork-flag, .hide-full-screen span.text-small', icon);
```
Comment (fregante) [inline source/features/ci-link.tsx:28]: The DOM completely changed, `.fork-flat` is not inside `h1`.

You can probably just use `select([itemprop="name"])!.after(icon)`

File: source/features/releases-tab.tsx [hunk 2]
```diff
@@ -59,7 +59,7 @@ async function init(): Promise<false | void> {
 		return false;
 	}
 
-	await elementReady('.pagehead + *'); // Wait for the tab bar to be loaded
+	await elementReady('.pagehead + *, .js-repo-nav + *'); // Wait for the tab bar to be loaded
```
Comment (fregante) [inline source/features/releases-tab.tsx:62]: Nothing exists after `.js-repo-nav`, but this should work
```suggestion
	await elementReady('.pagehead + *, .UnderlineNav-body + *'); // Wait for the tab bar to be loaded
```

File: source/features/show-open-prs-of-forks.tsx [hunk 3]
```diff
@@ -67,7 +67,7 @@ async function initHeadHint(): Promise<void | false> {
 		return false;
 	}
 
-	select('.fork-flag .text')!.append(
+	select('.fork-flag .text, .hide-full-screen span.text-small')!.append(
```
Comment (fregante) [inline source/features/show-open-prs-of-forks.tsx:70]: Same as before I think:

```suggestion
	select<HTMLAnchorElement>(`[data-hovercard-type="repository"][href="/${getForkSourceRepo()}"]`)!.append(
```

**Comment by fregante** [comment]
If supporting the old version gets messy, let's just drop it. GHE users will have to upgrade eventually. Such is life.

File: source/features/ci-link.tsx [hunk 4]
```diff
@@ -25,7 +25,7 @@ async function init(): Promise<false | void> {
 	}
 
 	// Append to title (aware of forks and private repos)
-	appendBefore('.pagehead h1', '.fork-flag', icon);
+	select('[itemprop="name"]')!.after(icon);
```
Comment (fregante) [inline source/features/ci-link.tsx:28]: This works, but it's slightly misaligned

Screen Shot 2020-07-17 at 19 13 03

**Pull request merged**